monkey-runner/plugin/src/main/groovy/com/novoda/monkey/MonkeyConfigurationPlugin.groovy

```diff
@@ -33,30 +33,33 @@ public class MonkeyConfigurationPlugin implements Plugin<Project> {
     private void configureTask(Project project, MonkeyRunnerExtension extension) {
         def runMonkeyAllTask = project.task(TASK_NAME)
 
-        def android = project.extensions.findByName("android")
+        def android = project.android
         android.command.devices().eachWithIndex { device, index ->
 
             def monkeyTask = project.task("runMonkeyDevice${index}", type: TargetedMonkey) {
                 packageName = extension.packageNameFilter
-                events = extension.eventsCount
+                adb = android.command.adb
```

Comment:
Or we were thinking about having a utility method in the plugin to attach `null` fields in the task. We haven't done that yet.